sechub-solution/README.adoc

```diff
@@ -14,6 +14,18 @@ To start a single container run the script:
 ./01-start-single-docker-compose.sh
 ----
 
+=== Single Container with Prepare (remote Data)
+
+. Adjust the `docker-compose_sechub-debiian-prepare.yml` file and replace <user> with your linux home directory. +
+Make sure the ./sechub/shared_volume folder exists and is writable for the docker. If not create the directory and execute `sudo chmod -R 777 ./sechub/sharedvolume`. +
+This mounted NFS must be the same directory as the NFS directory in the PDS prepare (/pds-solutions/prepare/docker-compose-prepare_external-network.yml.
```

Comment:
Sorry, but I do not really like that somebody has to replace pseudo parameter <user> if somebody wants to run sechub solution . I know we discussed that we could use user home location - but I was not aware that a user has to do something here manually.

SecHub shall always work "out of the box".

An alternative could be to use
https://docs.docker.com/reference/dockerfile/#arg and provide this by startup script to the docker startup phase.

But I think the easier way would be to 
- ensure /tmp/sechub-shared-volume in shell scripts (before startup)
- use this path in both parts as shared volume
- done

When this is done, the block inside the documentation should changed.